sema/check/base.py

```
@@ -0,0 +1,31 @@
+        self.type = type_test
+        self.options = options
+        self.result = CheckResult(
+            success=False,
+            error=False,
+            message="",
+            url=url,
+            type_test=type_test,
+        )
+
+    @abstractmethod
+    def run(self) -> CheckResult:
```

Ruff

Missing type annotation for `self` in method

(ANN101)